### PR TITLE
1.5.0-insiders changelog

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
Did we want to wait to get any other fixes in?

**Comment by sean-mcmanus** [comment]
Looks like we're waiting for one more fix...

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
 
+## Version 1.5.0-insiders: June 14, 2021
+### Enhancements
+* Add "Symbol Options" for CppVsdbg to configure symbol settings [PR #7680](https://github.com/microsoft/vscode-cpptools/pull/7680)
+* Update CppVsdbg to use newer CppEE and msdia. 
+
+### Bug Fixes
+* Fix Find All References reporting certain references in headers as inactive. [#7609](https://github.com/microsoft/vscode-cpptools/issues/7609)
+* Fix IntelliSense process crash and tag parser failure with columns > 65535. [#7621](https://github.com/microsoft/vscode-cpptools/issues/7621)
+* Fix incorrect translations.
```
Comment (michelleangela) [inline Extension/CHANGELOG.md]: nit: change to "Fix incorrect localization translations." so it's less ambiguous.

Note: already updated this in internal copy on OneNote.

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+  * jogo- (@jogo-) [PR #7625](https://github.com/microsoft/vscode-cpptools/pull/7625)
+* Fix `autocompleteAddParentheses` for some template argument deduction cases. [#7626](https://github.com/microsoft/vscode-cpptools/issues/7626)
+* Fix some incorrect IntelliSense errors. [#6639](https://github.com/microsoft/vscode-cpptools/issues/6639], [#7630](https://github.com/microsoft/vscode-cpptools/issues/7630)
+* Change references to OS X to macOS.
```
Comment (michelleangela) [inline Extension/CHANGELOG.md]: There should be an "of" --> "Change references of "OS X" to macOS."

Note: already updated this in internal copy on OneNote.

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Good catch

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: You mean translation could be interpreted as a non-localization translation, i.e. like a geometric translation?

Comment (michelleangela) [reply Extension/CHANGELOG.md]: or any kind translation, like I was thinking translation unit initially. So suggested to add "localization" to clarify.

Comment (bobbrow) [reply Extension/CHANGELOG.md]: They were French translations, so you could say "French translations" to remove any ambiguity.

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: I think some were portugese too in another change.

**Review by michelleangela — APPROVED** [review]

**Pull request merged**